README.md

```diff
@@ -0,0 +1,92 @@
+
+O objetivo é documentar quais são as preocupações principais envolvidas em administrar o grupo, como realizar algumas ações técnicas tais como postar no site ou outros meios de comunicação, organizar um contest, entre outros.
+
+Este documento está em constante desenvolvimento.
+
+## Princípios
+
+### Proatividade
+
+É importante manter sempre em mente a ideia de que esse grupo, para se manter funcionando, não pode depender de uma pessoa só. Ele tem que funcionar como grupo e tem que contar com a colaboração das pessoas envolvidas. Isso é essencial porque:
+
+1. As pessoas tem que treinar além de organizar o grupo.  
+2. As pessoas, em algum momento, se formam, vão fazer estágio, entre outros e o grupo não deveria parar na ausência de certas pessoas.
+3. Criar um ambiente onde as pessoas tem liberdade para contribuir e são responsáveis pelos seus próprios interesses inclui pessoas que estão a fim de contribuir e levar o grupo adiante.
+
+### Continuidade
+
+O grupo deve pensar em como vai continuar funcionando após a progressão das gerações do próprio grupo. Isso faz com que seja essencial se preocupar em recrutar novas pessoas e incentivar que o máximo de pessoas participem.
+
+## Coach
+
+Pera. Mas não é o Coach que toca o grupo? Não. O Coach, idealmente, deveria se preocupar com as seguintes tarefas:
+
+1) Organizar a [seletiva](#seletiva)
+2) Organizar as [reuniões](#reuniões), **especialmente**, a reunião de começo de ano.
+3) Inscrever os times
+
+Naturalmente, Coaches costumam se propor a fazer mais, como, idealmente, organizar contests, montar listas interessantes, reservar salas de treino ou aulas, entre outros.
+
+## Reuniões
+
+Uma diretiva geral que vem funcionando bem para reuniões de um grupo que funciona tão distribuidamente é a de delegar responsabilidades. A ideia funciona assim:
+
+- Cada proposta tem um responsável. Esta pessoa fica incumbida de fazer com que a proposta seja encaminhada.
+- Seguindo o princípio de [proatividade](#proatividade), o responsável deve se voluntariar a ter essa tarefa.
+- Se ninguém se voluntaria, a tarefa não vai ser supervisionada e a proposta não vai pra frente. Só vão ser carregadas propostas com pessoas interessadas em realizá-las.
+
+### Reunião de começo de ano
+
+Alguns assuntos devem ser decididos o quanto antes no ano, idealmente, durante uma reunião. Algumas questões que certamente devem ser discutidas todo ano são:
+- Como vai funcionar o [recrutamento](#recrutamento)?
+- Quem vai fazer a [seletiva](#seletiva)?
+- Como vão ser os [treinos](#treinos)?
+- Como vão ser as [reuniões mensais](#reunioes-mensais)?
+- Quem vai organizar o [bixeCamp](#bixecamp)?
+- Quem vai organizar o [simulado de bixes](#simulado-de-bixes)?
+- Como vão funcionar as [aulas avançadas](#aulas-avancadas)?
+- Como vai funcionar o [acampamento](#acampamento)?
+- Onde vai ser o japonês depois da primeira reunião?
+
```

Comment:
Adicionaria a questão de como fazer as camisetas

Reply:
Boua gá-ro-to. Merchan em geral né? Camisetas, adesivos e sabe-se lá o que mais.

Reply:
Verdade, mas pode ficar para o próximo PR.